framework/src/wifi_manager/wifi_manager.c

```diff
@@ -1027,7 +864,7 @@ wifi_manager_result_e _handler_on_uninitialized_state(_wifimgr_msg_s *msg)
 	wifi_utils_info_s info;
 	wifi_utils_result_e wres = wifi_utils_get_info(&info);
 	if (wres != WIFI_UTILS_SUCCESS) {
-		ndbg("[WM] wifi_utils_get_info fail");
+		ndbg("[WM] T%d wifi_utils_get_info fail\n", getpid());
```

Comment:
Is "T" before %d required? If so, what does that mean..?

Reply:
'T' means "thread number", which is widely used in the code.